Assets/Tests/EditMode/FlightPlanTests.cs

```diff
@@ -0,0 +1,46 @@
+using NUnit.Framework;
+using UnityEngine;
+
+public class FlightPlanTests {
+  [Test]
+  public void Waypoints_ReturnsWaypointsSortedByDescendingDistance() {
+    var flightPlanConfig = new Configs.FlightPlan() {
```

Comment:
nit: prefer explicit/strong types

Reply:
The type is already given by the constructor on the RHS